docs/UserGuide.md

```diff
@@ -96,6 +96,33 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567`
 
+Do take note that there is a near duplicate check for names. This means that when 
+adding a person whose name is similar to another client already found in DonnaFin, 
+the result display will prompt the user as such but the client will still be added to 
+the contact list. However, if the names are complete duplcates they will be rejected.
+
+Near duplicate:
+![DuplicateRejectionMessage](images/NearDuplicateRejectionMessage.png)
+
+Duplicate:
+![FullDuplicateRejectionMessage](./images/FullDuplicateRejectionMessage.png)
+
+Names are said to be near duplicates if the lowercase of both names are the same or when
+a space in the name is input as 2 or more spaces. See below for more examples
+
+| First Name     | Second Name                        | Is near duplicate
+| -------------- | ---------------------------------------| -------------------------------------------------------------------------------------------------- |
+| david Li     | David LI                        | yes
+| daviD Li     | David LI                        | yes
+| daVid Li     | David LI                        | yes
+| david Li     | David    &nbsp; LI                        | yes
+| david Li     | David     &nbsp;  &nbsp; &nbsp;     LI              | yes
+| davidLi     | David Li                        | no
+| DavidLi     | David Li                        | no
+
+Note that this does not apply for near duplicates for 
+no spaces and one space as seen in the last 2 examples above.
```

Comment:
Confusing leh. How's this:

Note that this duplicate detection does not try to combine separated words. E.g. 'John Son' and 'Johnson' would not be seen as near duplicates.

Reply:
thats some nice phrasing